src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { filterSlice } from '../features/filter';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
```

Comment:
The function 'combineSlices' does not exist in '@reduxjs/toolkit'. You likely meant to use 'combineReducers'. Please replace 'combineSlices' with 'combineReducers'.